app/Knights/knight_class.py

```diff
@@ -0,0 +1,20 @@
+from typing import List, Dict, Optional
```

Comment:
Rename your module to use lower-letter naming in a single form.

Reply:
👍